cmd/ciam-rebac/wire.go

```diff
@@ -17,5 +19,5 @@ import (
 
 // wireApp init kratos application.
 func wireApp(*conf.Server, *conf.Data, log.Logger) (*kratos.App, func(), error) {
-	panic(wire.Build(server.ProviderSet, service.ProviderSet, newApp))
+	panic(wire.Build(server.ProviderSet, service.ProviderSet, biz.ProviderSet, data.ProviderSet, newApp))
```

Comment:
You don't have to make change here, 

you can refer: https://github.com/RedHatInsights/insights-rebac/blob/main/internal/service/service.go
for adding provider

Reply:
Is that the best place? I thought the pattern was to add a ProviderSet at each layer, and then add all of the sets in wireApp with wire.Build. (But I'm new to wire.go, so I could be wrong.)

Reply:
Yes, that's what I mean by adding a provider, adding an ad for each layer. I think did correctly here, got confused.